ui/app/src/components/PublishDialog/PublishDialogContainer.tsx

```diff
@@ -83,68 +83,70 @@ export function PublishDialogContainer(props: PublishDialogContainerProps) {
       publishingTarget: null
     };
 
-    let itemsChecked = items.filter((item) => selectedItems[item.path]);
+    if (detailedItems) {
```

Comment:
This section is a memo that used the items (Dialog prop) to set the scheduled value (and some other values for the component) that is used for the now/custom setting. We already fetch the detailedItems in this component, so now the calculations are done using them, instead of using the items prop (that may be incomplete Sandbox items).

Reply:
Make sure all the logic still applies correctly when there are mixed items (e.g with and without schedule)

Reply:
Tested and it does work with mixed items